includes/formatting.php

```diff
@@ -544,16 +544,17 @@ function give_format_decimal( $amount, $dp = false, $sanitize = true ) {
  *
  * @since 1.0
  *
- * @param string $price The donation amount.
- * @param string $currency The currency code.
- * @param bool   $decode_currency Whether to decode the currency HTML format or not.
+ * @param string         $price           The donation amount.
+ * @param string         $currency        The currency code.
+ * @param bool           $decode_currency Whether to decode the currency HTML format or not.
+ * @param string|integer $form_id         Donation Form ID.
  *
  * @return mixed|string
  */
-function give_currency_filter( $price = '', $currency = '', $decode_currency = false ) {
+function give_currency_filter( $price = '', $currency = '', $decode_currency = false, $form_id = '' ) {
```

Comment:
@emgk Can we convert param into single array with backward compatibility

Reply:
Sure, I will do it on Monday.  @ravinderk